app/gui2/src/components/visualizations/ScatterplotVisualization.vue

```diff
@@ -539,12 +539,13 @@ useEvent(document, 'keydown', bindings.handler({ zoomToSelected: () => zoomToSel
 <template>
   <VisualizationContainer :belowToolbar="true">
     <template #toolbar>
-      <button class="image-button active">
-        <SvgIcon name="show_all" alt="Fit all" @click.stop="zoomToSelected(false)" />
-      </button>
-      <button class="image-button" :class="{ active: brushExtent != null }">
-        <SvgIcon name="find" alt="Zoom to selected" @click.stop="zoomToSelected" />
-      </button>
+      <SvgButton name="show_all" alt="Fit all" @click.stop="zoomToSelected(false)" />
+      <SvgButton
+        name="find"
+        alt="Zoom to selected"
+        :class="{ disabled: brushExtent == null }"
```

Comment:
Disabled state should rather be a prop than just a class. That could then influence the actual [disabled](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/button#disabled) button attribute.